hasura/migrations/default/1695665224892_alter_table_public_profiles_add_column_product_emails/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."profiles" add column "product_emails" boolean
+ not null default 'true';
```

Comment:
Still think we might want this to be false but we can address that when we actually set it in the UI or improve the verify flow